jahia-test-module/src/react/server/views/testAreaColumns/TestAreaColumns.tsx

```diff
@@ -10,10 +10,10 @@ jahiaComponent(
     return (
       <div data-testid={`row-${currentNode.getName()}`}>
         <div data-testid={`${currentNode.getName()}-col-1`}>
-          <Area name={`${currentNode.getName()}-col-1`} areaAsSubNode={true} />
+          <Area name={`${currentNode.getName()}-col-1`} />
         </div>
         <div data-testid={`${currentNode.getName()}-col-2`}>
-          <Area name={`${currentNode.getName()}-col-2`} areaAsSubNode={true} />
+          <Area name={`${currentNode.getName()}-col-2`} />
```

Comment:
Does this work as expected? Are `node-col-1/2` correctly attached as children of the current node?

Reply:
yes, it uses (and creates if it does not exist) the node identified by the `name` as a child node of the current page